validphys2/src/validphys/tests/test_plots.py

```diff
@@ -41,6 +41,21 @@ def test_dataspecschi2():
     )
 
 
+@pytest.mark.linux
+@pytest.mark.mpl_image_compare(tolerance=TOLERANCE_VALUE)
+def test_plotfancy():
+    return next(
+        iter(
+            API.plot_fancy(
+                dataset_input={'dataset': 'NMC_NC_NOTFIXED_P_EM-SIGMARED', 'variant': 'legacy'},
+                theoryid=THEORYID,
+                pdfs=[PDF],
+                use_cuts='internal',
```

Comment:
Could you use one of the data configs from comftest.py? (perhaps with minimal modifications)

Reply:
Yes, I can do that. This was first a test to check if what I generated locally could match the CI....and as always I seem to have really different architecture/configurations from the CI...

Reply:
I think we are even skipping the matplotlib tests in mac because of how hard it was to make them compatible with linux...

Reply:
But the weird thing is that it is passing on macOS while failing on ubuntu 😅 I will investigate later.